hardware/Schematic Design Guide.md

```diff
@@ -0,0 +1,148 @@
+
+  Sub-G for:  Proprietary, Z-Wave.
+
+- Output power levels: 
+
+  0dBm, +8dBm, +10dBm, +13~14dBm, +20dBm.
+- DC-DC regulator configuration: 
+
+  Use it for power efficiency; 
+ 
+  Don’t use it for saving cost.
+
+- Chip packages:
+
+  32/48/68 pins QFN package and 125pins BGA package for EFR32 chips.
+
+For example, you want to search the reference schematic of proprietary chip, you can go to the following web page:
+
+![](../files/HW-Schematic-Design-Guide/Proprietary-Page.png)
+In this page, you can move mouse to Products tab, then the pop-down menu will appear with Wireless products which include the Proprietary item, then click left button of your mouse. And it will enter the Proprietary product list page as following:
+![](../files/HW-Schematic-Design-Guide/Proprietary-Product.png)
+You can find the generations of proprietary products as listed in the bottom table. You can select EFR32FG1 series 1 sub-GHz and 2.4GHz SoCs for more details:
+![](../files/HW-Schematic-Design-Guide/Proprietary-S1.png)
+In this page, you can find the brief introduction of the proprietary series 1 chips and the parameter list of each chips. Ignore the chips list, you can click the View Document in this page and acess the documents list pages directly:
+![](../files/HW-Schematic-Design-Guide/Proprietary-Reference.png)
+For example, If you want to look for 434MHz +10dBm output power reference design, you can click item EFR32FG1 BRD4251B 2400/433 MHz 10dBm Radio Board Full Design Package and download the reference design package to your local address.
+The schematic likes:[BRD4251B](https://www.silabs.com/documents/public/schematic-files/EFR32FG1-BRD4251B-B00-schematic.pdf)
+
+But keep in mind that this reference design can be applied to a cluster of chips that have difference packages and frequency band configurations. So you can just replace the chips to your interest chip with different package and frequency band.
+## Hardware design considerations
+After you get the reference design, you may have the concern why the reference is designed like this. Therefore Silabs provides a lot of application notes or reference manual for customer reference. For basical hardware considerations, Customer can refer to the following application note for detail descriptions:[AN0002.1](https://www.silabs.com/documents/public/application-notes/an0002.1-efr32-efm32-series-1-hardware-design-considerations.pdf). And there are several hardware considerations in different divisions as following:
+- Power supply configurations: 
+  - General requirements: VREGVDD = AVDD, must be the highest voltage on  EFR32; DVDD,IOVDD,RFVDD,PAVDD =< AVDD; DECOUPLE < DVDD
+  - If internal DC-DC is not used, typically tie all the power pins to the main power supply.
+  - If internal DC-DC is used, typically tie VREGVDD, AVDD, IOVDD to the main power supply, tie DVDD, RFVDD to DCDC output (VREGSW), tie PAVDD to DCDC output if TXP =< +13dBm, tie PAVDD to VMCU main supply if TXP > +13dBm.
```

Comment:
Shall we add an explanation about the term "TXP" ? 
e.g
TXP (Transmitting Power)